src/cloud/components/Modal/contents/TemplatesModal/index.tsx

```diff
@@ -487,6 +485,7 @@ const TemplatesModal = ({ callback }: TemplatesModalProps) => {
 
 const TemplatesModalContainer = styled.div`
   height: 80vh;
+  margin: -12px 0;
```

Comment:
do not use negative margins.
openModal function has options. 

If you want to remove the padding on the modals' contents
instead of using 

```
openModal(<TemplatesModal />, { width: 'large' })
```

use
 
```
openModal(<TemplatesModal />, { width: 'large', removePadding: true })
```

Reply:
Fixed